### PR TITLE
ThreatConnect: various changes from testing and customer feedback

### PR DESCRIPTION
**Pull request by GShepherdTC**
Enhancement

## Proposed commit message

Remove default SSL certificate, update descriptions, add ability to control attributes, associated groups and indicators. Changes came from internal testing and customer feedback.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] New radio sliders added during setup wizard
- [ ]  Removed default SSL cert
- [ ]  Documentation changes
- [ ]  Removed some fields from being added to ES

## How to test this PR locally

Load integration and review the changes.

## Related issues

- 

## Screenshots

-

### CODE REVIEW

File: packages/ti_threatconnect/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.6.0"
+  changes:
+    - description: Remove default SSL certificate, update descriptions, add ability to control attributes, associated groups and indicators.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/9829
```
Comment (efd6) [inline packages/ti_threatconnect/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/9836
```

File: packages/ti_threatconnect/data_stream/indicator/manifest.yml [hunk 2]
```diff
@@ -1,19 +1,44 @@
...
+        description: Filter results based on query written in [TQL](https://knowledge.threatconnect.com/docs/constructing-query-expressions).
         multi: false
         required: false
         show_user: true
+      - name: include_attributes
+        type: bool
+        title: Include Attributes in the response
+        multi: false
+        required: false
+        show_user: false # Drives if the input is shown at top and not under advanced settings
```
Comment (efd6) [inline packages/ti_threatconnect/data_stream/indicator/manifest.yml:23]:
```suggestion
        show_user: false # Drives whether the input is shown at top and not under advanced settings.
```

File: packages/ti_threatconnect/elasticsearch/transform/latest/fields/fields.yml
[the remark this answers is not in the document]
Comment (kcreddy) [reply packages/ti_threatconnect/elasticsearch/transform/latest/fields/fields.yml:1]: That is correct. The only field that shouldn't exist in this transform's `fields.yml` is the `labels.is_ioc_transform_source`. This field should only be present in datastream's `fields.yml` file.

[the remark this answers is not in the document]
Comment (GShepherdTC) [reply packages/ti_threatconnect/elasticsearch/transform/latest/fields/fields.yml:1]: Not sure I fully follow.  Is there a suggested change then?   Or are we good?

[the remark this answers is not in the document]
Comment (efd6) [reply packages/ti_threatconnect/elasticsearch/transform/latest/fields/fields.yml:1]: They need to be in sync mod the field that @kcreddy mentioned.

**Comment by efd6** [comment]
@GShepherdTC I've push the required changes for the fields definitions in order to get the tests passing. If this has added fields that you don't want, please remove them from both fields.yml files and adjust the test inputs to remove the fields.

**Review by efd6 — APPROVED** [review]
This LGTM now, but will wait for @kcreddy.

**Review by kcreddy — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**